### PR TITLE
updates with SHKs CD-64

### PR DESCRIPTION
**Pull request by oscaryee19**
adjusted the holdings example to include SHK
https://finbourne.atlassian.net/browse/CD-64

### CODE REVIEW

File: examples/lusid/holdings/2-create-and-inline-txn-property.sql [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+%%luminesce
+@keysToCatalog = values
+('Transaction/luminesce-examples/strategy', 'strategy', True, 'A property representing SHK');
+
+@config = select column1 as [Key], column2 as Name, column3 as IsMain, column4 as Description from @keysToCatalog;
+
+select * from Sys.Admin.Lusid.Provider.Configure
+where Provider = 'Lusid.Portfolio.Holding.Writer'
+and Configuration = @config;
```
Comment (slemasne) [inline examples/lusid/holdings/2-create-and-inline-txn-property.sql:51]: Explicitly set this to Modify. See support page.

File: examples/lusid/holdings/.DS_Store
Comment (slemasne) [inline examples/lusid/holdings/.DS_Store:1]: What are these .DS files?
Remove them

**Review by slemasne — COMMENTED** [review]
2 small changes

File: examples/lusid/holdings/2-create-and-inline-txn-property.sql (hunk 1 above)
Comment (oscaryee19) [reply examples/lusid/holdings/2-create-and-inline-txn-property.sql]: done

File: examples/lusid/holdings/.DS_Store
Comment (oscaryee19) [reply examples/lusid/holdings/.DS_Store:1]: done

**Review by slemasne — APPROVED** [review]

**Pull request merged**